### PR TITLE
add fastqc performance post

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+site: freiburg
+tags: [devops, tools]
+title: FASTQC jobs and very long reads
+location: Galaxy Europe
+---
+
+Over the last weeks we saw many [FASTQC]({{ page.website }}/?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72) jobs running for days or even longer, which is not normal.
+We investigated this and saw that all of those jobs are processing files with unusal long reads.
```
Comment (mtekman) [inline _posts/2019-04-14-fastqc-long-reads.md]: unusal → unusually

Comment (gmauro) [inline _posts/2019-04-14-fastqc-long-reads.md]: replace unusal with unusual

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+However, FASTQC seems to have a problem with that so we updated the tool and tried to understand what is going on.
+We finally found that FASTQC has a hard-coded parameter (-Xmx250m) to restrict the Java virtual machine to a certain amount of memory.
+Increasing this value made all our tests way more faster with very long reads and we hope to speed up your computation from days to minutes.
+
+We will kill long running jobs that do not finish in a few days, all what you need to do is restarting the job and it should finish way faster.
+Please get in [contact](mailto:contact@usegalaxy.eu) with us, if you still see such problems with FASTQC.
+
+Please also have a look at [fastp]({{ page.website }}/?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/) as an alternative tool, that also
+provides some quality metrices.
```
Comment (gmauro) [inline _posts/2019-04-14-fastqc-long-reads.md]: replace metrices with metrics

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+tags: [devops, tools]
+title: FASTQC jobs and very long reads
+location: Galaxy Europe
+---
+
+Over the last weeks we saw many [FASTQC]({{ page.website }}/?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72) jobs running for days or even longer, which is not normal.
+We investigated this and saw that all of those jobs are processing files with unusal long reads.
+
+We are excited to see that our instance is used more and more for processing data of the
+fourth-generation of DNA sequencing technology!
```
Comment (mtekman) [inline _posts/2019-04-14-fastqc-long-reads.md]: → used more and more for processing data from fourth-generation DNA sequencing technology!

File: _posts/2019-04-14-fastqc-long-reads.md (hunk 1 above)
Comment (simonbray) [inline _posts/2019-04-14-fastqc-long-reads.md]: unusually

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+However, FASTQC seems to have a problem with that so we updated the tool and tried to understand what is going on.
+We finally found that FASTQC has a hard-coded parameter (-Xmx250m) to restrict the Java virtual machine to a certain amount of memory.
+Increasing this value made all our tests way more faster with very long reads and we hope to speed up your computation from days to minutes.
```
Comment (simonbray) [inline _posts/2019-04-14-fastqc-long-reads.md]: much faster

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+We will kill long running jobs that do not finish in a few days, all what you need to do is restarting the job and it should finish way faster.
```
Comment (simonbray) [inline _posts/2019-04-14-fastqc-long-reads.md]: what -> that, restarting -> restart

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+Over the last weeks we saw many [FASTQC]({{ page.website }}/?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72) jobs running for days or even longer, which is not normal.
+We investigated this and saw that all of those jobs are processing files with unusually long reads.
+
+We are excited to see that our instance is used more and more for processing data of the
+fourth-generation of DNA sequencing technology!
+
+However, FASTQC seems to have a problem with that so we updated the tool and tried to understand what is going on.
+We finally found that FASTQC has a hard-coded parameter (-Xmx250m) to restrict the Java virtual machine to a certain amount of memory.
+Increasing this value made all our tests way more faster with very long reads and we hope to speed up your computation from days to minutes.
```
Comment (mtekman) [inline _posts/2019-04-14-fastqc-long-reads.md]: all our tests → all of our tests
more faster with very long reads → much faster for very long reads

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+We will kill long running jobs that do not finish in a few days, all what you need to do is restarting the job and it should finish way faster.
+Please get in [contact](mailto:contact@usegalaxy.eu) with us, if you still see such problems with FASTQC.
+
```
Comment (simonbray) [inline _posts/2019-04-14-fastqc-long-reads.md]: delete such

File: _posts/2019-04-14-fastqc-long-reads.md (hunk 2 above)
Comment (simonbray) [inline _posts/2019-04-14-fastqc-long-reads.md]: metrices -> metrics

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+We investigated this and saw that all of those jobs are processing files with unusually long reads.
+
+We are excited to see that our instance is used more and more for processing data of the
+fourth-generation of DNA sequencing technology!
+
+However, FASTQC seems to have a problem with that so we updated the tool and tried to understand what is going on.
+We finally found that FASTQC has a hard-coded parameter (-Xmx250m) to restrict the Java virtual machine to a certain amount of memory.
+Increasing this value made all our tests way more faster with very long reads and we hope to speed up your computation from days to minutes.
+
+We will kill long running jobs that do not finish in a few days, all what you need to do is restarting the job and it should finish way faster.
```
Comment (mtekman) [inline _posts/2019-04-14-fastqc-long-reads.md]: days, all what you need to do is restarting the job → days, with all that is required of you is to restart the job

File: _posts/2019-04-14-fastqc-long-reads.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+---
+site: freiburg
+tags: [devops, tools]
+title: FASTQC jobs and very long reads
+location: Galaxy Europe
+---
+
+Over the last weeks we saw many [FASTQC]({{ page.website }}/?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72) jobs running for days or even longer, which is not normal.
```
Comment (mtekman) [inline _posts/2019-04-14-fastqc-long-reads.md]: last weeks → past few weeks

**Comment by bgruening** [comment]
Danke @simonbray @mtekman and @gmauro!

**Pull request merged**